phoenixRest/views/user/instance/__init__.py

```diff
@@ -496,8 +496,8 @@ def upload_avatar(context, request):
     # Crop the image
     im = im.crop((x, y, x+w, y+h))
 
-    # Deal with RGBA
-    if im.mode in ('RGBA', 'LA') or (im.mode == 'P' and 'transparency' in im.info):
```

Comment:
won't things crash on save even if the mode is P but no transparency?

Reply:
I thought the error was that PIL can't save a P image to JPEG(you have to convert it to RGB first)

Reply:
The mode being P is what implies transparency here, converting and then saving an image sith P mode is perfectly fine. As for the transparency key in the info part, none of the images i have parsed have contained this flag and i am not sure where you have gotten this from?

Reply:
> I thought the error was that PIL can't save a P image to JPEG(you have to convert it to RGB first)

Yes the code already does this, this is not the issue here

Reply:
https://pillow.readthedocs.io/en/stable/handbook/concepts.html

According to pillow docs, P is just "Palette" and PA is "Palette with alpha".

Reply:
I see, i'll try making a PA mode PBG

Reply:
transparent_p.png is not transparent. You can double check this with `exiftool`.